app.py

```diff
@@ -4,6 +4,34 @@
 from serializers import serialize_book
 
 app = Flask(__name__)
+app.config['JSON_AS_ASCII'] = False
+
+
+def put_update_book(connection, book_id):
+    body = request.json
+    new_book_name = body["name"]
+    cursor = connection.cursor()
+
+    # Check updating book in the db
+    response = cursor.execute(f'select name from books where id = {book_id}')
```

Comment:
it is better not to do like this, better to modify DB and set unique flag to the column.
Otherwise you PUT HTTP request will do one extra DB request which can affect performance:
Pls see reference here: https://www.sqlitetutorial.net/sqlite-unique-constraint/